Packages/FeatureServices/AppService/OnstartService.swift

```diff
@@ -47,9 +66,45 @@ public struct OnstartService: Sendable {
         } catch {
             debugLog("migrations error: \(error)")
         }
-        
+
         if !preferences.hasCurrency, let currency = Locale.current.currency {
             preferences.currency = (Currency(rawValue: currency.identifier) ?? .usd).rawValue
         }
     }
+
+    private func configureURLCache() {
+        URLCache.shared.memoryCapacity = 256_000_000 // ~256 MB memory space
+        URLCache.shared.diskCapacity = 1_000_000_000 // ~1GB disk cache space
+    }
+
+    private func excludeDirectoriesFromBackup() {
```

Comment:
```suggestion
    private func excludeDirectoriesFromBackup() throws {
```

handle errors on higher level